src/components/tour-modal/tour-modal-custom/tour-modal-custom.tsx

```diff
@@ -0,0 +1,51 @@
+import { Step, ModalName } from '../tour-modal';
+import step1Bilde from './step-1.png';
+import filter1Bilde from './filter-1.png';
+import filter2Bilde from './filter-2.png';
+import filter3Bilde from './filter-3.png';
+
+const stepsLastNedCV: Step[] = [
```

Comment:
Føler denne konfigen burde ligge i nærheten av hvor endringlogg konfigen ligger.
Eller er tanken at de to skal være separate funksjoner som kan brukes uavhengig av hverandre?